src/modules/dataman/dataman.cpp

```diff
@@ -260,18 +148,18 @@ static enum {
 
 /* The data manager work queues */
 
-typedef struct {
+struct work_q_t {
 	sq_queue_t q;		/* Nuttx queue */
-	px4_sem_t mutex;		/* Mutual exclusion on work queue adds and deletes */
-	unsigned size;		/* Current size of queue */
-	unsigned max_size;	/* Maximum queue size reached */
-} work_q_t;
+	px4_sem_t mutex;	/* Mutual exclusion on work queue adds and deletes */
+	uint8_t size{0};		/* Current size of queue */
+	uint8_t max_size{0};	/* Maximum queue size reached */
+};
 
 static work_q_t g_free_q;	/* queue of free work items. So that we don't always need to call malloc and free*/
 static work_q_t g_work_q;	/* pending work items. To be consumed by worker thread */
 
 static px4_sem_t g_work_queued_sema;	/* To notify worker thread a work item has been queued */
-static px4_sem_t g_init_sema;
```

Comment:
Also here.

Reply:
I've restored the static. In file and ram initialize they now call `dm_init_post()` instead.